CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(http-parser VERSION 2.8.0)
+
+add_library(http_parser "http_parser.c" "http_parser.h")
+target_compile_definitions(http_parser PRIVATE
```

Comment:
`http_parser.h` installed and there is a check for `HTTP_PARSER_STRICT` there, hence it should be `PUBLIC` instead of `PRIVATE`.

Reply:
Though it's not really used there. In general I think it's better to move this check:
* https://github.com/hunter-packages/http-parser/blob/hunter-2.8.0/http_parser.h#L48-L53

to `*.c` file.

Reply:
I was thinking based on the original Makefile, it has to be private. Otherwise, the project would have generated a header or something for dependent libraries.